backend/services/wallet/endpoints.md

```diff
@@ -0,0 +1,60 @@
+# Wallet API
+
+## Create a new wallet for a user
+
+> `POST` <http://localhost/wallet/create>
+
+**Required JSON:**
+
+```json
+{
+  "public_key": string
+}
+```
+
+Registers a wallet for a new user in the blockchain and returns the server message.
+
+**Example return:**
+
+```json
+201 
+{
+  "success": boolean
```

Comment:
I think here you can just use `true` as I don't think `false` will ever appear in the success path, so we can be explicit